pyintegration/deephaven2/table.py

```diff
@@ -1001,8 +1001,31 @@ def agg_by(self, aggs: List[Aggregation], by: List[str]) -> Table:
             DHError
         """
         try:
-            j_agg_list = j_array_list([agg.j_agg for agg in aggs])
+            j_agg_list = j_array_list([agg.j_aggregation for agg in aggs])
             return Table(j_table=self.j_table.aggBy(j_agg_list, *by))
         except Exception as e:
             raise DHError(e, "table agg_by operation failed.") from e
+
+    def agg_all_by(self, agg: Aggregation, by: List[str]) -> Table:
+        """ The agg_all_by method creates a new table containing grouping columns and grouped data. The resulting
+        grouped data is defined by the aggregation specified.
+
+        Note, because agg_all_by() applies the aggregation to all the columns of the table, it will ignore
+        any column names specified for the aggregation
```

Comment:
```suggestion
        Note, because agg_all_by applies the aggregation to all the columns of the table, it will ignore
        any column names specified for the aggregation.
```
I removed the parentheses because `agg.py` does not have them.  I don't know what we have been doing as a standard.